### PR TITLE
added changes

### PR DESCRIPTION
**Pull request by rubelux**

### CODE REVIEW

File: src/components/CreatorBox.tsx [hunk 1]
```diff
@@ -5,34 +5,32 @@ import { breakpoint, palette } from '@theme'
 import { rgba, sizeForLevel } from '@lib'
 
 interface ICreatorBox {
-  member: IUserPublicFragment
+  user: IUserPublicFragment
```
Comment (EricWVGG) [inline src/components/CreatorBox.tsx:8]: thank you for clarifying user vs member, that's been messy

File: src/components/SettingsParts/EditSettings/index.tsx [hunk 2]
```diff
@@ -60,8 +59,7 @@ const FormWrapper = styled.div`
   display: grid;
   gap: 10px;
   grid-template-areas:
-    'firstName lastName'
-    'email email'
+    'artizenHandle'
```
Comment (EricWVGG) [inline src/components/SettingsParts/EditSettings/index.tsx:62]: this should be `'artizenHandle artizenHandle'` so it spans two columns.

File: src/lib/sendArtifactToIPFS/index.ts [hunk 3]
```diff
@@ -19,7 +19,7 @@ interface MetadataObject {
 
 const sendArtifactToIPFS = async (artifactNumber: number, season: ISeasonFragment, project: IProjectFragment) => {
   const artifactName = `Artifact #${artifactNumber}`
-  const allProjectMembersString = project.members.map(({ user }) => `${user?.firstName} ${user?.lastName}`).join(', ')
+  const allProjectMembersString = project.members.map(({ user }) => `${user?.artizenHandle}`).join(', ')
```
Comment (EricWVGG) [inline src/lib/sendArtifactToIPFS/index.ts:22]: we don't need to `.join()`

**Review by EricWVGG — APPROVED** [review]

File: src/components/CreateProfile/lib/index.ts [hunk 4]
```diff
@@ -88,24 +97,22 @@ const useCreateProfile = (initialFormState: FormState | FormStateAdmin) => {
...
+    console.log('user is created::::', newUser)
+    // If there is a change in email, or artizenHandle, update profile in Courier
+    if (newUser.email && newUser.artizenHandle) {
+      await sendUserToCourier(newUser.id, newUser.email, newUser.artizenHandle)
     }
     return newUserMutationReturn.data.insert_Users.returning[0]
   }
 
-  const updateProfile = async (userIdToUpdate: string) => {
+  const updateProfile = async (userIdToUpdate: string, sendWelcomeEmail: boolean) => {
```
[the remark this answers is not in the document]
Comment (rubelux) [reply src/components/CreateProfile/lib/index.ts:109]: I believe it's optional, it only sends the email when users complete their profile:

```
if (status === 'authenticated' && !!loggedInUser && (!loggedInUser.email || !loggedInUser.artizenHandle)) {
      setVisibleModalWithAttrs('createProfile', {
        action: 'update',
        sendWelcomeEmail: true,
      })
    }
```

[the remark this answers is not in the document]
Comment (rubelux) [reply src/components/CreateProfile/lib/index.ts:109]: but happy to change it if you mean something different

**Review by EricWVGG — APPROVED** [review]

**Pull request merged**